linux/linux.mk

```diff
@@ -35,6 +35,7 @@ ifeq "$(ALLOW_STATIC)" "yes"
 
 else
 $(eval	$(call ADD_Lib,libturbojpeg))
+	LDD_LIBS += -limobiledevice-1.0
```

Comment:
Since different distro name these packages differently (Ubuntu is the standout), I think the correct approach is to define an input variable, like this: https://github.com/dev47apps/droidcam-linux-client/blob/master/Makefile#L18, and use that with the eval call.

Reply:
Upon second inspection here as well I noticed that on ArchLinux the usbmuxd package is dependent on libimobiledevice and since the source is already dependent on usbmuxd there is no need for me to explicitly set the direct dependency on libimobiledevice so it appears this is also superfluous.   I'll update my pull request removing both changes to linux/linux.mk.

Reply:
When I remove ```LD_LIBS += -limobiledevice-1.0``` from the linux/linux.mk make file I see the following error in OBS's log:

```
13:11:02.195: os_dlopen(/usr/lib/obs-plugins/droidcam-obs.so->/usr/lib/obs-plugins/droidcam-obs.so): /usr/lib/obs-plugins/droidcam-obs.so: undefined symbol: lockdownd_client_new
13:11:02.195: 
13:11:02.198: os_dlopen(/usr/lib/obs-plugins/droidcam-obs.so->/usr/lib/obs-plugins/droidcam-obs.so): /usr/lib/obs-plugins/droidcam-obs.so: undefined symbol: lockdownd_client_new
13:11:02.198: 
13:11:02.198: Module '/usr/lib/obs-plugins/droidcam-obs.so' not loaded
```